graphite_api/utils.py

```diff
@@ -90,6 +90,15 @@ def to_seconds(delta):
     return abs(delta.seconds + delta.days * 86400)
 
 
+def deltaseconds(timedelta):
+    """
+    Convert a timedelta object into seconds
+    (same as timedelta.total_seconds() in Python 2.7+)
```

Comment:
I guess we can.  Python 2.6 has been failing for some time because of dropped support in third party dependencies anyhow.